src/taxpasta/infrastructure/cli/standardise.py

```diff
@@ -158,6 +158,12 @@ def standardise(
         help="Add the taxon's entire lineage to the output. These are taxon "
         "identifiers separated by semi-colons.",
     ),
+    add_rank_lineage: bool = typer.Option(  # noqa: B008
+        False,
+        "--add-rank-lineage",
+        help="Add the taxon's entire rank lineage to the output. These are taxon "
+        "ranks separated by semi-colons.",
```

Comment:
Correct this help string is split across two lines?

Reply:
It will render as a continuous string. Just split in source code to respect the line length.